services/FileCreate.proto

```diff
@@ -33,7 +33,7 @@ After the file is created, the FileID for it can be found in the receipt, or rec
 
  The file contains the specified contents (possibly empty). The file will automatically disappear at the expirationTime, unless its expiration is extended by another transaction before that time. If the file is deleted, then its contents will become empty and it will be marked as deleted until it expires, and then it will cease to exist.
 
- The keys field is a list of keys. All the keys on the list must sign to create or modify a file, but only one of them needs to sign in order to delete the file.  Each of those "keys" may itself be threshold key containing other keys (including other threshold keys). In other words, the behavior is an AND for create/modify, OR for delete. This is useful for acting as a revocation server. If it is desired to have the behavior be AND for all 3 operations (or OR for all 3), then the list should have only a single Key, which is a threshold key, with N=1 for OR, N=M for AND.
+ The keys field is a list of keys. All keys within the top level key list must sign (M-M) to create or modify a file, but only one key (1-M) is required to sign from the top level key list in order to delete the file.  Each of those "keys" may itself be threshold key containing other keys (including other threshold keys). In other words, the behavior is an AND for create/modify, OR for delete. This is useful for acting as a revocation server. If it is desired to have the behavior be AND for all 3 operations (or OR for all 3), then the list should have only a single Key, which is a threshold key, with N=1 for OR, N=M for AND.
```

Comment:
hate to be pedantic, but I would try to explain what is a "top level key". I would also try to avoid a run on sentence by putting the requirement for deletion in a separate sentence:

_All keys within the top level key list must sign (M-M) to create or modify a file. However, in order to delete the file, only one key (1-M) is required to sign from the top level key list._